lib/clif-backend/Cargo.toml

```diff
@@ -9,13 +9,14 @@ edition = "2018"
 
 [dependencies]
 wasmer-runtime-core = { path = "../runtime-core", version = "0.4.2" }
-cranelift-native = "0.30.0"
-cranelift-codegen = "0.30.0"
-cranelift-entity = "0.30.0"
-cranelift-wasm = "0.30.0"
+cranelift-native = { git = "https://github.com/wasmerio/cranelift.git", branch = "wasmer" }
+cranelift-codegen = { git = "https://github.com/wasmerio/cranelift.git", branch = "wasmer" }
+cranelift-entity = { git = "https://github.com/wasmerio/cranelift.git", branch = "wasmer" }
+cranelift-frontend = { git = "https://github.com/wasmerio/cranelift.git", branch = "wasmer" }
+cranelift-wasm = { git = "https://github.com/wasmerio/cranelift.git", branch = "wasmer" }
```

Comment:
We will probably need to update https://github.com/wasmerio/cranelift to the latest commits in cranelift